src/less/radio/ds6/radio-base.less

```diff
@@ -100,26 +47,26 @@
 }
 
 .radio__icon {
-    color: @ds6-color-g204-gray;
+    color: @ds6-color-g206-gray;
```

Comment:
Could this be `inherit`? Or are we saying we have to add a modifier to change the checkbox color?

Reply:
Yeah, it *could* be. But if, for whatever reason, the app wants to use the default inline SVG icons, they would then no longer be the default gray color. They would have to specify that color themselves. But that might be fine. I'm open to changing it.

Reply:
And yes, right now they have to add a modifier to change the color. That need would go away if we make the change, with caveat, above.

Reply:
I've been trying to let colors cascade as much as possible. So, in this case, whatever the text color is, the checkbox would follow suit. Additionally, they could put a `<span>` around it and style it however they want.

Reply:
I think we can do this, given that icon color is the same as default text color:

```less
@ds6-color-g206-gray: #111820;
@ds6-color-text-default: @ds6-color-g206-gray;
```

Reply:
We would still need the `--inherit` modifier, though.

Reply:
Why? The only rule it has is `color: inherit`. I thought you wanted to move that rule to `.radio__icon`?

Reply:
I think we're speaking past each other. Let me try and clarify:

I'm in favor of changing the rule inside of `radio__icon` to be `color: inherit;`, and removing the modifier.

I thought you were saying we'd change it to `color: @ds6-color-text-default;`, which prompted me to say that we'd still need the modifier.

Reply:
Ok, I made the change. @PrashantAshok: when @seangates approves, please duplicate the changes in this PR in checkbox module.